sql-statements/sql-statement-create-table.md

```diff
@@ -258,7 +258,7 @@ mysql> DESC t1;
 * The `[ASC | DESC]` in `index_col_name` is currently parsed but ignored (MySQL 5.7 compatible behavior).
 * The `COMMENT` attribute does not support the `WITH PARSER` option.
 * TiDB supports 1017 columns in a single table by default and 4096 columns at most. The corresponding number limit in InnoDB is 1017 columns, and the hard limit in MySQL is 4096 columns. For details, see [TiDB Limitations](/tidb-limitations.md).
-* For partitioned tables, only Range, Hash and Range Columns (single column) are supported. For details, see [partitioned table](/partitioned-table.md).
+* TiDB supports `HASH`, `RANGE`, `LIST`, and `KEY` partitioning types. For an unsupported partition type, TiDB returns `Warning: Unsupported partition type %s, treat as normal table`, where `%s` is the specific unsupported partition type.
```

Comment:
this updated sentence is from https://docs.pingcap.com/tidb/dev/mysql-compatibility#ddl-operations